app/src/components/Map/ControlPanel.jsx

```diff
@@ -139,15 +161,11 @@ class ControlPanel extends Component {
 
   renderResume() {
     return (
-      <div className={controlPanelStyle['resume-display']}>
-        <div className={controlPanelStyle['categories-display']}>
-          <div className={controlPanelStyle['vessel-display']}>
-            <span className={controlPanelStyle['counter-description']}>vessels</span>
-            <span className={controlPanelStyle.total}>224,654</span>
-          </div>
-          <div className={controlPanelStyle['activity-display']}>
-            <span className={controlPanelStyle['counter-description']}>activity</span>
-            <span className={controlPanelStyle.total}>224,654</span>
+      <div className={controlPanelStyle['resume-display']} >
+        <div className={controlPanelStyle['categories-display']} >
+          <div className={controlPanelStyle['vessel-display']} >
+            <span className={controlPanelStyle['counter-description']} >vessels</span>
```

Comment:
I updated the json file to have large int values - something like "number of vessels" or "number of fishing events". Anyway, this is not yet defined, so let's not worry too much about it until we know exactly what we'll have here